src/reference/antora/modules/ROOT/pages/ip/tcp-connection-factories.adoc

```diff
@@ -194,6 +194,26 @@ As noted there, such modifications are possible if SSL is being used, or not.
 
 Also see xref:ip/annotation.adoc[Annotation-Based Configuration] and xref:ip/dsl.adoc[Using the Java DSL for TCP Components].
 
+[[host-verification]]
+== Host verification
+Starting from version 5.1.0, host verification is enabled by default for enhanced security. This feature ensures that the server's identity is verified during TCP connections.
```

Comment:
Please, add a new line after the title.
We also prefer to follow rule `One sentence per line`: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
please check. Should be resolved.